### PR TITLE
Updating the main Readme.md

### PR DESCRIPTION
**Pull request by andreiborodin**
Updating the main readme following introductions of samples.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
 # MSBuildForUnity
 
-MSBuildForUnity is a [collection of tools](#features) to help integrate MSBuild with Unity. This is helpful in leveraging more of the .NET ecosystem within Unity. Some things this project helps with are:
-- Sharing code (via an MSBuild project) between Unity and other .NET projects (such as UWP XAML apps, Xamarin apps, etc.).
+MSBuildForUnity solves the problem of establishing a clear dependency relationships between Unity project and other .NET components such as external (to Unity) C# projects, or NuGet packages. It creates a familiar to .NET developers project structure and ensures that the dependencies are resolves and brought into the Unity project as appropriate. With this component, you can:
```
Comment (keveleigh) [inline README.md]:
```suggestion
MSBuildForUnity solves the problem of establishing a clear dependency relationship between Unity projects and other .NET components such as external (to Unity) C# projects, or NuGet packages. It creates a familiar to .NET developers project structure and ensures that the dependencies are resolved and brought into the Unity project as appropriate. With this component, you can:
```

Comment (andreiborodin) [reply README.md]: I feel we could go either way, I will choose single "Unity project" and it's numerous (possibly different types) "dependency relationships"

Comment (ryantrem) [inline README.md]: "resolve**s**" should be "resolve**d**"

Comment (keveleigh) [reply README.md]: This is fixed in #35

**Pull request merged**